bowtie/report/src/index.js

```diff
@@ -0,0 +1,25 @@
+import { createRoot } from "react-dom/client";
+import App from "./App";
+import { CountsDataProvider } from "./data/CountsDataContext";
+
+document.addEventListener("DOMContentLoaded", () => {
+  const Dialect = "2020-12";
+
+  //fetching json data from url
+  fetch(`https://bowtie-json-schema.github.io/bowtie/draft${Dialect}.json.gz`)
```

Comment:
Is there a way to use data generated by local bowtie?

Reply:
Yes of course! The raw data can be found here (in compressed form) ----
`https://bowtie-json-schema.github.io/bowtie/draft2020-12.json.gz` (just replace the **_dialect_** version of your choice)

Reply:
Hmm, to be precise I meant e.g. using some command like this: `bowtie suite ... / bowtie report` and then using produced data by this command to populate UI

Reply:
Ohh! My bad! Do you mean populating the local UI or the hosted one?

Reply:
I mean local UI with local report data. Now the URL is hardcoded for specific host (https://bowtie-json-schema.github.io), maybe this URL can be relative so it could work locally too? Also, if in the future hosting platform gonna change then it shouldn't really affect code behaviour. So I would suggest something like `/bowtie/draft${Dialect}.json.gz`, but you gonna need to have these report files locally. And somehow expose them on live local server

Reply:
"We can definitely add conditional fetching of data based on where the app is being used, whether to fetch data from the **_remote URL_** or from a **_local path_**. Regarding the idea of using a **_relative URL_**, I believe we can address it in a separate pull request in the near future. @Julian, please let me know your thoughts on this.

Reply:
Yeah totally fine with me, @harrel56 we definitely want to preserve that behavior but yeah we can perhaps get this one merged and then restore it i in a next PR (I think being able to "upload" a local file is probably the most straightforward way to resupport it but we can brainstorm)